pkg/components/networkpolicy.go

```diff
@@ -0,0 +1,119 @@
+package components
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	netv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+)
+
+func GetNetworkPolices(
```

Comment:
note that now we install it always, we need a condition please for OpenShift as there it is not relevant
because else it means we always deploy something that doesnt fit

the right way is to not just have if OpenShift then not install, but to just install the right one please

Reply:
I removed this code following this comment https://github.com/kubevirt/cluster-network-addons-operator/pull/2294#discussion_r2100239810